### ISSUE TITLE
Model evaluation on test set - part 2

### ISSUE THREAD

**Issue by swenkel** [jhfjhfj1/autokeras#167]
### Bug Description
First of all: thank you for fast response to [part 1](https://github.com/jhfjhfj1/autokeras/issues/162). However, I noticed something new:

It seems like ```clf.evaluate``` returns ```1.0``` all the time. It solves the problem of returning different numbers but 1.0 acc would be great but is unlikely on fashion-mnist after 60 mins of training.

### Reproducing Steps
```python
y_pred_augmented = clf_augmented.evaluate(X_test, y_test)
print(y_pred_augmented)
```
returns ```1.0``` all the time

```python
from sklearn.metrics import accuracy_score
y_prediction_augmented = clf_augmented.predict(X_test)
accuracy_score(y_true=y_test, y_pred=y_prediction_augmented)
```
returns ```0.92``` all the time
### Expected Behavior

It should return ```0.92```.

### Setup Details
Include the details about the versions of:
 - OS type and version: Linux
 - Python: 3.6.5
 - autokeras: develop
 - scikit-learn: 0.19.1
 - numpy: 1.14.5
 - keras: 2.2.2
 - scipy: 1.1.0
 - tensorflow: 1.10.0
 - pytorch: 0.4.1

**Comment by jhfjhfj1** [comment]
@swenkel 
Thank you for your report!
The bug is fixed in the new release.
